playwright/playwright.config.ts

```diff
@@ -0,0 +1,71 @@
+import {resolve} from 'path';
+
+import type {PlaywrightTestConfig} from '@playwright/experimental-ct-react';
+import {defineConfig, devices} from '@playwright/experimental-ct-react';
+import react from '@vitejs/plugin-react';
+
+function pathFromRoot(p: string) {
+    return resolve(__dirname, '../', p);
+}
+
+const reporter: PlaywrightTestConfig['reporter'] = [];
+
+reporter.push(
+    ['list'],
+    [
+        'html',
+        {
+            open: process.env.CI ? 'never' : 'on-failure',
+            outputFolder: resolve(process.cwd(), 'playwright-report'),
+        },
+    ],
+);
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+    testDir: pathFromRoot('src'),
+    testMatch: '*/__tests__/*.visual.test.tsx',
+    updateSnapshots: process.env.UPDATE_REQUEST ? 'all' : 'missing',
+    snapshotPathTemplate:
+        '{testDir}/{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}-linux{ext}',
+    /* The base directory, relative to the config file, for snapshot files created with toMatchSnapshot and toHaveScreenshot. */
+    /* Maximum time one test can run for. */
+    timeout: 10 * 1000,
+    /* Run tests in files in parallel */
+    fullyParallel: true,
+    /* Fail the build on CI if you accidentally left test.only in the source code. */
+    forbidOnly: Boolean(process.env.CI),
+    /* Retry on CI only */
+    retries: 0,
```

Comment:
Shouldn't be there `process.env.CI` env?

Reply:
the tests work stably, of course you can make two retries

Reply:
I write that because you have comment above: `Retry on CI only`